src/subdomains/generic/gs/dto/db-query.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { IsDate, IsNotEmpty, IsNumber, IsOptional, IsString } from 'class-validator';
+
+export class DbQueryBaseDto {
+  @IsNotEmpty()
+  @IsString()
+  table: string;
+
+  @IsNotEmpty()
+  @IsNumber()
+  min = 1;
+
+  @IsOptional()
+  @IsNumber()
+  maxLine: number;
+
+  @IsNotEmpty()
+  @IsDate()
+  updatedSince: Date = new Date(0);
+
+  @IsNotEmpty()
+  @IsString()
+  sorting: 'ASC' | 'DESC' = 'ASC';
+
+  // Comma separated column names
+  @IsOptional()
+  select?: string[];
```

Comment:
Why is this `string[]`, if it is comma separated? I would go for string array, as we are using a body here.